packages/app/README.md

```diff
@@ -0,0 +1,69 @@
+# startupjs app
+> Router component for multi apps
+
+## Installation
+
+```sh
+yarn add @startupjs/app
+```
+
+## Codepush setup
+
+You should setup codepush as described
+[here](https://docs.microsoft.com/en-us/appcenter/distribution/codepush/react-native)
+and [here](https://paper.dropbox.com/doc/Setup-CodePush-for-RN-app.--AgoJbp6ktolYMqTE0ksiVBqTAg-nKoimxRHPVDhAV8xgUei5)
+
+## Usage
+
+### client
+
+```js
+import App from '@startupjs/app'
+import * as main from '../main'
+import * as admin from '../admin'
+
+// Object where key is Platform.OS and value is number
+const CRITICAL_VERSION = {
+  ios: Number,
+  android: Number,
+  web: Number
+}
+
+return (
+  <App
+    apps={{ main, admin }}
+    supportEmail='support@lingua.live'
+    criticalVersion=CRITICAL_VERSION
+    iosUpdateLink='itms://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=1451260061&mt=8'
+    androidUpdateLink='market://details?id=dm.somProject'
```

Comment:
move supportEmail and update links to constants on the top of the file, like CRITICAL_VERSION. And don't use`lingua.live` in the example, specify `admin@example.com`

Reply:
also fix indentation of iosUpdateLink, and the name of the android app has a typo. And overall just use generic names everywhere -- `id=company.example.app` and for ios id=0000000000&mt=8